src/icon-manifest/iconGenerator.ts

```diff
@@ -2,28 +2,19 @@ import * as fs from 'fs';
 import * as path from 'path';
 import * as _ from 'lodash';
 import { SettingsManager } from '../settings';
-import { pathUnixJoin, fileFormatToString } from '../utils';
-import {
-  IIconGenerator,
-  IVSCode,
-  IIconSchema,
-  ISettings,
-  IFolderCollection,
-  IFileCollection,
-  IDefaultExtension,
-  IIconPath,
-} from '../models';
+import * as utils from '../utils';
+import * as model from '../models';
```

Comment:
I would use `models` instead of `model` given that we usually  mimic the name of the module when we do a full import.